### PR TITLE
add hello.txt

### PR DESCRIPTION
**Pull request by carlopascual**

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello this is carlol!!
```
Comment (DPASCII) [inline hello.txt:1]: Are you sure?

Comment (carlopascual) [reply hello.txt:1]: Yeah pretty sure

Comment (DPASCII) [inline hello.txt:1]: I want 3 exclamations

**Review by DPASCII — CHANGES_REQUESTED** [review]

File: hello.txt [hunk 2]
```diff
@@ -1 +1 @@
-Hello this is carlol!!
\ No newline at end of file
+Hello this is carlol!!!
```
Comment (DPASCII) [inline hello.txt:1]: perf

**Review by DPASCII — APPROVED** [review]
perf gj boyo

**Pull request merged**